site/content/docs/developing/domain.en.md

```diff
@@ -50,7 +50,16 @@ nlb:
   alias: example-v1.aws
 ```
 
-## What happens under the hood?
+### How do I configure an alias with an existing domain?
+If you want to use alias for your existing domain (not the one when you create the application), you could specify `--import-cert-arns` flag when creating the environment to import validated ACM certificates that include the alias. For example:
+
+```
+$ copilot env init --import-cert-arns arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
+```
+
+After deploying a service to that environment, add the DNS of the Application Load Balancer (ALB) created in the environment as an A record to where your alias domain is hosted.
+
```

Comment:
This seems to be a less detailed explanation than the blog post - do you think it'd help if we put a link to the blog post here?